### PR TITLE
[funnel] remove funnel persons on data return

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- remove the people array on funnel return as it's unused now for clickhouse
*If this affects the frontend, include screenshots.*  

## How did you test this code?

*Please describe.*
- update tests

### CODE REVIEW

File: ee/clickhouse/queries/funnels/funnel.py [hunk 1]
```diff
@@ -85,14 +75,14 @@ def _format_single_funnel(self, result, with_breakdown=False):
...
 
-            serialized_result = self._serialize_step(step, total_people, relevant_people[0:100])
+            serialized_result = self._serialize_step(step, total_people, [])
             if cast(int, step.order) > 0:
+
                 serialized_result.update(
                     {
-                        "average_conversion_time": result[cast(int, step.order) + num_entities * 2 - 1],
-                        "median_conversion_time": result[cast(int, step.order) + num_entities * 3 - 2],
+                        "average_conversion_time": result[cast(int, step.order) + num_entities * 1 - 1],
```
Comment (hazzadous) [inline ee/clickhouse/queries/funnels/funnel.py:84]: What are these changes for?

File: ee/clickhouse/queries/funnels/funnel.py [hunk 2]
```diff
@@ -85,14 +75,14 @@ def _format_single_funnel(self, result, with_breakdown=False):
 
             if result and len(result) > 0:
                 total_people += result[step.order]
-                relevant_people += result[cast(int, step.order) + num_entities]
 
-            serialized_result = self._serialize_step(step, total_people, relevant_people[0:100])
+            serialized_result = self._serialize_step(step, total_people, [])
```
Comment (hazzadous) [inline ee/clickhouse/queries/funnels/funnel.py:79]: Is there any way we can signal why we're not passing in anything for people here? Maybe just a comment?

File: ee/clickhouse/queries/funnels/test/breakdown_cases.py [hunk 3]
```diff
@@ -134,9 +134,7 @@ def test_funnel_step_breakdown_event(self):
                         "name": "sign up",
                         "custom_name": None,
                         "order": 0,
-                        "people": [person2.uuid, person3.uuid]
-                        if Funnel == ClickhouseFunnel
-                        else [],  # backwards compatibility
+                        "people": [] if Funnel == ClickhouseFunnel else [],  # backwards compatibility
```
Comment (hazzadous) [inline ee/clickhouse/queries/funnels/test/breakdown_cases.py:137]: Just `[]`?

i.e. both if and else are the same

**Review by hazzadous — APPROVED** [review]
Some minor points, nothing blocking

Is this actually causing issues btw? One thing I'm not so keen on is the difference now between postgres and clickhouse.

File: ee/clickhouse/queries/funnels/funnel.py (hunk 1 above)
Comment (neilkakkar) [reply ee/clickhouse/queries/funnels/funnel.py:84]: We removed `n` rows from the result (people arrays), so the placement of others in the result changes

**Pull request merged**